packages/next-swc/crates/core/tests/fixture/react-server-components/server-graph/client-entry/input.js

```diff
@@ -1,29 +1,27 @@
 // This is a comment.
 
-"use strict";
+'use strict'
 
 /**
  * This is a comment.
  */
 
-"use client";
+'use client'
 
 // This is a comment.
 
-"random-directive";
+'random-directive'
 
-import "fs"
-
-"qwerty";
+import 'fs'
+;('qwerty')
```

Comment:
You may want to look for all `;('` in these test fixtures and make them prettier ignored and directive-style